src/mds/Server.cc

```diff
@@ -5944,6 +5944,46 @@ int Server::parse_layout_vxattr(string name, string value, const OSDMap& osdmap,
   return 0;
 }
 
+int64_t Server::make_float_max_byte_parseable(string val)
```

Comment:
this isn't just parsing floats but also ints, so in case an error arises due to some invalid suffix (let's say `1QiB`) then saying `make_float_max_byte_parseable failed to parse quota.max_bytes: strict_iecstrtoll: unit prefix not recognized` wouldn't make sense because the error is clearly not related to parsing floats as the func name suggests. (This is a nit but for brevity) IMO either rename this to a more generic name or just move the error logging back to `Server::parse_quota_vxattr`.

Reply:
Switched the name to `make_quota_max_byte_parseable()`